static/app/views/monitors/components/overviewTimeline/jobTickTooltip.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {JobTickTooltip} from 'sentry/views/monitors/components/overviewTimeline/jobTickTooltip';
+
+type StatusCounts = [ok: number, missed: number, timeout: number, error: number];
+
+export function generateEnvMapping(name: string, counts: StatusCounts) {
```

Comment:
starting to have some second thoughts about copy pasting this everywhere now haha...

Reply:
hahahah